src/NzbDrone.Core.Test/Datastore/Migration/117_add_webrip_qualites_in_profileFixture.cs

```diff
@@ -0,0 +1,94 @@
+            {
+                c.Insert.IntoTable("Profiles").Row(new
+                {
+                    Id = 0,
+                    Name = "SDTV",
+                    Cutoff = 1,
+                    Items = $"[{GenerateQualityJson(1, true)}, {GenerateQualityJson((int)Quality.WEBRip480p, false)}, {GenerateQualityJson((int)Quality.WEBRip720p, false)}, {GenerateQualityJson((int)Quality.WEBRip1080p, false)}, {GenerateQualityJson((int)Quality.WEBRip2160p, false)} ]"
+                });
+            });
+
+            var profiles = db.Query<Profile116>("SELECT Items FROM Profiles LIMIT 1");
+
+            var items = profiles.First().Items;
+            items.Should().HaveCount(5);
+            items.Select(v => v.Quality).Should().BeEquivalentTo(1, null, null, null, null);
+            items.Select(v => v.Items.Count).Should().BeEquivalentTo(0, 2, 2, 2, 2);
+            items.Select(v => v.Allowed).Should().BeEquivalentTo(true, false, false, false, false);
+        }
+
+        [Test]
+        public void should_add_webrip_and_webdl_if_webdl_is_missing()
+        {
+            var db = WithMigrationTestDb(c =>
+            {
+                c.Insert.IntoTable("Profiles").Row(new
+                {
+                    Id = 0,
+                    Name = "SDTV",
+                    Cutoff = 1,
+                    Items = $"[{GenerateQualityJson(1, true)}, {GenerateQualityJson((int)Quality.WEBRip480p, false)}, {GenerateQualityJson((int)Quality.WEBRip720p, false)}, {GenerateQualityJson((int)Quality.WEBRip1080p, false)} ]"
+                });
+            });
+
+            var profiles = db.Query<Profile116>("SELECT Items FROM Profiles LIMIT 1");
+
+            var items = profiles.First().Items;
+            items.Should().HaveCount(5);
+            items.Select(v => v.Quality).Should().BeEquivalentTo(1, null, null, null, null);
+            items.Select(v => v.Items.Count).Should().BeEquivalentTo(0, 2, 2, 2, 2);
+            items.Select(v => v.Allowed).Should().BeEquivalentTo(true, false, false, false, false);
+        }
+
+        [Test]
+        public void should_group_webrip_and_webdl_with_the_same_resolution()
+        {
+            var db = WithMigrationTestDb(c =>
+            {
+                c.Insert.IntoTable("Profiles").Row(new
+                {
+                    Id = 0,
+                    Name = "SDTV",
+                    Cutoff = 1,
+                    Items = $"[{GenerateQualityJson(1, true)}, {GenerateQualityJson((int)Quality.WEBRip480p, false)}, {GenerateQualityJson((int)Quality.WEBRip720p, false)}, {GenerateQualityJson((int)Quality.WEBRip1080p, false)}, {GenerateQualityJson((int)Quality.WEBRip2160p, false)} ]"
+                });
+            });
+
+            var profiles = db.Query<Profile116>("SELECT Items FROM Profiles LIMIT 1");
+            var items = profiles.First().Items;
+
+            items[1].Items.First().Quality.Should().Be((int)Quality.WEBRip480p);
```

Comment:
Yeah, it is, the UI shows bottom up, not top down, so we want the lower quality first.